### ISSUE TITLE
player2.screenwavemedia.com without fallback

### ISSUE THREAD

**Issue by Meneth32** [rg3/youtube-dl#6270]
player2.screenwavemedia.com has been updated again. The previous fix forced a fallback to the old server, but that now redirects right back to player2. Output from 2015.07.18:
```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'http://teamfourstar.com/video/stream-four-star-j-stars-victory-vs/']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2015.07.18
[debug] Git HEAD: b0bff54
[debug] Python version 3.4.2 - Windows-7-6.1.7601-SP1
[debug] exe versions: ffmpeg N-50911-g9efcfbe
[debug] Proxy map: {}
[TeamFour] stream-four-star-j-stars-victory-vs: Downloading webpage
[ScreenwaveMedia] TeamFourStar-55a8d6ef23c58: Downloading player webpage
ERROR: Unable to extract vidtitle; please report this issue on https://yt-dl.org/bug . Make sure you are using the lates
t version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and inclu
de its complete output.
Traceback (most recent call last):
  File "youtube_dl\YoutubeDL.py", line 656, in extract_info
    ie_result = ie.extract(url)
  File "youtube_dl\extractor\common.py", line 275, in extract
    return self._real_extract(url)
  File "youtube_dl\extractor\screenwavemedia.py", line 29, in _real_extract
    r'\'vidtitle\'\s*:\s*"([^"]+)"', playerdata, 'vidtitle').replace('\\/', '/')
  File "youtube_dl\extractor\common.py", line 557, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract vidtitle; please report this issue on https://yt-dl.org/bug . Mak
e sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl wit
h the --verbose flag and include its complete output.
```

**Comment by Meneth32** [comment]
VLC is able to dump the stream if you give it an URL like this:
http://206.217.201.111/vod/smil:TeamFourStar-55ad26bc14c5a.smil/TeamFourStar-55ad26bc14c5a__hd1.mp4_chunk.m3u8?nimblesessionid=7635839

The correct URL for your stream can be found using the AdBlock addon ("Open blockable items").

**Comment by jaimeMF** [comment]
Fixed by @remitamine in #6330, thanks for the report.
